test/test-0018.sh

```diff
@@ -5,34 +5,38 @@
 cleanup 18
 
 # ------------------------------- Test 18 ------------------------------------
-preptest test.log 18 1
+needle="$(uuidgen -r)"
```

Comment:
I don't think we need to be too worried about availability of `uuidgen` when `journalctl` is used in the same script. To be honest `test-0018.sh` should be flagged as linux only.

Survival of results is about journal entries that are system wide and span over separate test runs, sessions, and so on. In short the grep in test needs to have something uniq to match for each check run, else results can be ambiguous.